test/PublicApiTests/BaseLine/Microsoft.OData.PublicApi.net45.bsl

```diff
@@ -7498,74 +7470,6 @@ public sealed class Microsoft.OData.UriParser.Aggregation.GroupByTransformationN
-    Warning = 2
-}
-
-public abstract class Microsoft.OData.UriParser.Validation.ODataUrlValidationRule {
-    protected ODataUrlValidationRule ()
-
-    bool IncludeImpliedProperties  { public get; public set; }
-    string RuleName  { public get; protected set; }
-
-    internal abstract System.Type GetRuleType ()
-    internal abstract void Validate (Microsoft.OData.UriParser.Validation.ODataUrlValidationContext context, object component)
-}
-
-[
-ExtensionAttribute(),
-]
-public sealed class Microsoft.OData.UriParser.Validation.ODataUrlValidationExtensions {
-    [
-    ExtensionAttribute(),
-    ]
-    public static bool ValidateODataUrl (System.Uri uri, Microsoft.OData.Edm.IEdmModel model, Microsoft.OData.UriParser.Validation.ODataUrlValidationRuleSet rules, out System.Collections.Generic.IEnumerable`1[[Microsoft.OData.UriParser.Validation.ODataUrlValidationMessage]]& validationMessages)
-}
-
-public sealed class Microsoft.OData.UriParser.Validation.ODataUrlValidationMessageCodes {
-    public static string DeprecatedElement = "deprecated"
-    public static string InvalidRule = "invalidRule"
-    public static string MissingSelect = "missingSelect"
-    public static string UnableToParseUri = "unableToParseUrl"
-}
-
-public sealed class Microsoft.OData.UriParser.Validation.ODataUrlValidationRules {
-    public static Microsoft.OData.UriParser.Validation.ODataUrlValidationRule DeprecatedNavigationSourceRule = Microsoft.OData.UriParser.Validation.ODataUrlValidationRule`1[Microsoft.OData.Edm.IEdmNavigationSource]
-    public static Microsoft.OData.UriParser.Validation.ODataUrlValidationRule DeprecatedPropertyRule = Microsoft.OData.UriParser.Validation.ODataUrlValidationRule`1[Microsoft.OData.Edm.IEdmProperty]
-    public static Microsoft.OData.UriParser.Validation.ODataUrlValidationRule DeprecatedTypeRule = Microsoft.OData.UriParser.Validation.ODataUrlValidationRule`1[Microsoft.OData.Edm.IEdmType]
-    public static Microsoft.OData.UriParser.Validation.ODataUrlValidationRule RequireSelectRule = Microsoft.OData.UriParser.Validation.ODataUrlValidationRule`1[Microsoft.OData.ODataUri]
-}
-
-public class Microsoft.OData.UriParser.Validation.ODataUrlValidationMessage {
-    public ODataUrlValidationMessage (string code, string message, Microsoft.OData.UriParser.Validation.Severity severity)
-
-    System.Collections.Generic.Dictionary`2[[System.String],[System.Object]] ExtendedProperties  { public get; }
-    string Message  { public get; }
-    string MessageCode  { public get; }
-    Microsoft.OData.UriParser.Validation.Severity Severity  { public get; }
-}
-
-public sealed class Microsoft.OData.UriParser.Validation.ODataUrlValidationContext {
-    System.Collections.Generic.List`1[[Microsoft.OData.UriParser.Validation.ODataUrlValidationMessage]] Messages  { public get; }
-    Microsoft.OData.Edm.IEdmModel Model  { public get; }
-
-    public void AddError (string code, string message, Microsoft.OData.UriParser.Validation.Severity severity)

```

Comment:
>AddError [](start = 16, length = 8)

Should be AddMessage

Reply:
Fixed in Issue #1828

---
In reply to: [445656103](https://github.com/OData/odata.net/pull/1826#discussion_r445656103) [](ancestors = 445656103)